src/views/virtualmachines/details/tabs/configuration/storage/components/tables/disk/DiskRow.tsx

```diff
@@ -60,6 +63,11 @@ const DiskRow: FC<
 
   const hasPVC = isPVCSource(obj);
 
+  const disks = getDisks(vm) || [];
+  const disk = disks.find((d) => d.name === name);
+  const isCDROM = disk && isCDROMDisk(disk);
+  const displayName = isCDROM ? t('cd-rom') : name;
```

Comment:
string to constant, I also think it should exist somewhere in the project please check